Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	$ cd ./gnark_backend_ffi; \
+		go build -o libgnark_backend.so -buildmode=c-shared ./main.go
+	$ cargo build
+
+test:
+	$ DYLD_LIBRARY_PATH=./gnark_backend_ffi cargo test
```

Comment:
Add a new line here. Try to update vscode's config so it'd add it for you in case you forget it.